pitop/robotics/pan_tilt_controller.py

```diff
@@ -0,0 +1,12 @@
+from pitop.pma import ServoMotor
+from pitop.system.port_manager import PortManager
+
+
+class PanTiltController:
+
+    def __init__(self, servo_pan_port="S0", servo_tilt_port="S3"):
+        self._pan_servo = ServoMotor(servo_pan_port)
+        self._tilt_servo = ServoMotor(servo_tilt_port)
+        self.__port_manager = PortManager()
+        self.__port_manager.register_component_instance(self._pan_servo, servo_pan_port)
```

Comment:
This hasn't got my changes from `roboticsKit2` branch, which I thought was appropriate. Should we try and get it in?